main.cpp

```diff
@@ -388,7 +368,35 @@ Thing 5) Recording Room
     2) provides energy consumption
     3) allows communication with the mix room
  */
-
+struct recRoom
+{
+  int instruments;
+  double airConditionerBill;
+  std::string roomName;
+  float sizeOfRoom;
+  int numbOfCable;
+
+  struct drumSet
+  {  
+    drumSet();
+
+    std::string brand = "DW";
+    std::string model = "Frequient Flyer";
+    int kickSize = 20;
+    int snareSize = 14;
+    int floorTomSize = 14;
+
+    void kickTheKick(int bpm);
+    void closeTheHihat(int bpm);
+    void kickTheSnare(int bpm);
+  };
+
+  void recordDrum(float sampleRate = 44.1f, int bitRate = 16);
+  void openTheLight();
+  void communicateToMixRoom(drumSet drumBeats);
+
+  drumSet played;
+};
```

Comment:
Fix the following:
- Type Name capitalization
- Indentation relative to the `{ }`
- [ ] revise